app/engine/Flywheel.test.js

```diff
@@ -0,0 +1,342 @@
+'use strict'
+/*
+  Open Rowing Monitor, https://github.com/jaapvanekris/openrowingmonitor
+*/
+import { test } from 'uvu'
+import * as assert from 'uvu/assert'
+import { deepMerge } from '../tools/Helper.js'
+import { replayRowingSession } from '../tools/RowingRecorder.js'
+import rowerProfiles from '../../config/rowerProfiles.js'
+
+import { createFlywheel } from './Flywheel.js'
+
+const baseConfig = {
+  numOfImpulsesPerRevolution: 6,
+  smoothing: 1,
+  flankLength: 11,
+  numberOfErrorsAllowed: 2,
+  minimumStrokeQuality: 0.30,
+  minumumRecoverySlope: 0,
+  autoAdjustRecoverySlope: true,
+  autoAdjustRecoverySlopeMargin: 0.10,
+  minumumForceBeforeStroke: 50,
+  minimumRecoveryTime: 2,
+  minimumTimeBetweenImpulses: 0.005,
+  maximumTimeBetweenImpulses: 0.02,
+  autoAdjustDragFactor: true,
+  dragFactorSmoothing: 3,
+  dragFactor: 100,
+  minimumDragQuality: 0.83,
+  flywheelInertia: 0.1,
+  sprocketRadius: 2
+}
+
+// Test behaviour for no datapoints
+test('Correct Flywheel behaviour at initialisation', () => {
+  const flywheel = createFlywheel(baseConfig)
+  testDeltaTime(flywheel, 0)
+  testSpinningTime(flywheel, 0)
+  testAngularPosition(flywheel, 0)
+  testAngularVelocity(flywheel, 0)
+  testAngularAcceleration(flywheel, 0)
+  testTorque(flywheel, 0)
+  testDragFactor(flywheel, 0.0001)
+  testIsDwelling(flywheel, false)
+  testIsUnpowered(flywheel, false)
+  testIsPowered(flywheel, true)
+})
+
+// Test behaviour for one datapoint
```

Comment:
Are these placeholders for additional tests that we should add in the future? In that case add a `todo: add test case for ...` so we don't miss them when elaborating. If it means something else it would be good if you could make the comment more meaningful.